src/__tests__/App.test.js

```diff
@@ -0,0 +1,10 @@
+import { render, screen } from "@testing-library/react";
+import App from "../components/App";
+
+describe('App', () => {
+  const { asFragment } = render(<App />)
+
+  test("app renders correctly", () => {
+    expect(asFragment).toMatchSnapshot()
```

Comment:
If you look at the snapshot generated by this test, you should expect to see a layout of a component complete with divs, text rendered, classnames etc, but currently it doesnt look like that. This is because you need to call the `asFragment` method in the expect statement:
```
    expect(asFragment()).toMatchSnapshot()
```
If you change this as re run the test, the existing snapshot will fail and it will prompt you to inspect the changes and update the snapshot or the component accordingly. In this case you want to update the snapshot by pressing `u`